config/application.rb

```diff
@@ -42,6 +42,7 @@ class Application < Rails::Application
     # Tells Rails to serve error pages from the app itself, rather than using static error pages in public/
     config.exceptions_app = routes
 
+    config.middleware.use Rack::Deflater
```

Comment:
This has no impact on existing requests. It just provides additional functionality for those requests that include compression headers (specifically only gzip)